test/framefusion.test.ts

```diff
@@ -380,7 +410,7 @@ describe('FrameFusion', () => {
         // Act & assert
         // ensure we render the last few frames properly - we have to flush the decoder to get the last few frames
         for (let i = 50; i < 60; i++) {
-            const time = i / FPS + FRAME_SYNC_DELTA;
```

Comment:
FRAME_SYNC_DELTA guarantee us that we don't seek to boundary of frame.
Because with this PR I'm fixing seeking to boundary, I decided delete FRAME_SYNC_DELTA

Reply:
Yeah that's a good change!